Bricks/strings/printf.h

```diff
@@ -33,6 +33,9 @@ SOFTWARE.
 namespace current {
 namespace strings {
 
+#ifndef CURRENT_WINDOWS
```

Comment:
Oh, I'm an idiot. This is *not* for Windows.

Perhaps guard it by compiler type/version then? An `#ifdef` for whether we're under `g++` or `clang++`?

CC @mzhurovich